src/main/java/Duke.java

```diff
@@ -1,10 +1,93 @@
+            if (line.equals("list")) {
+                for (int i = 0; i < taskNumber; i++) {
+                    System.out.println((i + 1) + "." + tasks[i]);
+                }
+            } else if (line.startsWith("done")) {
+                int index = Integer.parseInt(line.substring(5)) - 1;
+                tasks[index].setDone();
+                System.out.println("Nice! I've marked this task as done:");
+                System.out.println("  " + tasks[index].getStatusIcon() + " " + tasks[index].getDescription());
+            } else if (line.startsWith("todo")) {
+                tasks[taskNumber] = new ToDo(line.substring(5));
+                taskNumber++;
+                System.out.println("Got it. I've added this task:");
+                System.out.println("  " + tasks[taskNumber - 1]);
+                System.out.println("Now you have " + taskNumber + " tasks in the list");
+            } else if (line.startsWith("deadline")) {
+                String[] words = line.split(" ");
+                int index = 0;
+                String deadlineDescription = "";
+                String by = "";
+                for (int i = 0; i < words.length; i++) {
+                    if (words[i].equals("/by")) {
+                        index = i;
+                        break;
+                    }
+                }
+                for (int i = 1; i < index; i++) {
+                    deadlineDescription = deadlineDescription + words[i] + " ";
+                }
+                by = words[index + 1];
+                tasks[taskNumber] = new Deadline(deadlineDescription, by);
+                taskNumber++;
+                System.out.println("Got it. I've added this task:");
+                System.out.println("  " + tasks[taskNumber - 1]);
+                System.out.println("Now you have " + taskNumber + " tasks in the list");
+            } else if (line.startsWith("event")) {
+                String[] words = line.split(" ");
+                int index = 0;
+                String eventDescription = "";
+                String at = "";
+                for (int i = 0; i < words.length; i++) {
+                    if (words[i].equals("/at")) {
+                        index = i;
+                        break;
+                    }
+                }
+                for (int i = 1; i < index; i++) {
+                    eventDescription = eventDescription + words[i] + " ";
+                }
+                for (int i = index + 1; i < words.length; i++) {
+                    at = at + words[i] + " ";
+                }
+                tasks[taskNumber] = new Event(eventDescription, at);
+                taskNumber++;
+                System.out.println("Got it. I've added this task:");
+                System.out.println("  " + tasks[taskNumber - 1]);
+                System.out.println("Now you have " + taskNumber + " tasks in the list");
+            }
+        }
+    }
```

Comment:
Method is over 30 lines of code. It would not fit into the computer screen making it hard to debug and read. Maybe split the method into smaller methods.